editor-service/src/main/java/com/qreal/robots/database/diagrams/model/LinkProperty.java

```diff
@@ -0,0 +1,97 @@
+package com.qreal.robots.database.diagrams.model;
+
+import com.qreal.robots.thrift.gen.TProperty;
+import java.io.Serializable;
+
+/**
+ * Property of a link.
+ */
+public class LinkProperty implements Serializable {
```

Comment:
is there a single reason why we need two separate but identical classes for node property and link property?